jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/openshift/AutoTLSEnricherTest.java

```diff
@@ -91,18 +91,18 @@ private AdaptTestConfig(PlatformMode mode, String initContainerNameConfig, Strin
     @Test
     public void testAdapt() throws Exception {
         final AdaptTestConfig[] data = new AdaptTestConfig[] {
-                new AdaptTestConfig(PlatformMode.kubernetes, null, null, null, null, null, null, null, null),
-                new AdaptTestConfig(PlatformMode.openshift, null, "tls-jks-converter", null,
+                new AdaptTestConfig(RuntimeMode.KUBERNETES, null, null, null, null, null, null, null, null),
+                new AdaptTestConfig(RuntimeMode.OPENSHIFT, null, "tls-jks-converter", null,
                         "jimmidyson/pemtokeystore:v0.1.0", null, "tls-pem", null, "tls-jks"),
-                new AdaptTestConfig(PlatformMode.openshift, null, "tls-jks-converter", null,
+                new AdaptTestConfig(RuntimeMode.OPENSHIFT, null, "tls-jks-converter", null,
                         "jimmidyson/pemtokeystore:v0.1.0", "tls-a", "tls-a", null, "tls-jks"),
-                new AdaptTestConfig(PlatformMode.openshift, null, "tls-jks-converter", null,
+                new AdaptTestConfig(RuntimeMode.OPENSHIFT, null, "tls-jks-converter", null,
                         "jimmidyson/pemtokeystore:v0.1.0", null, "tls-pem", "jks-b", "jks-b"),
-                new AdaptTestConfig(PlatformMode.openshift, "test-container-name", "test-container-name", "image/123",
+                new AdaptTestConfig(RuntimeMode.OPENSHIFT, "test-container-name", "test-container-name", "image/123",
                         "image/123", "tls-a", "tls-a", "jks-b", "jks-b") };
 
         for (final AdaptTestConfig tc : data) {
-            TreeMap configMap = new TreeMap() {
+            TreeMap<String, String> configMap = new TreeMap<String, String>() {
```

Comment:
```suggestion
            TreeMap<String, String> configMap = new TreeMap<>() {
```

Reply:
I tried doing this but I couldn't use '<>' with anonymous inner classes. Our project is at JDK8 right now, maybe we should upgrade to JDK11? WDYT?

Reply:
We should not be doing this, this is really bad.
Creating an inner class extending a TreeMap is something I don't see really the purpose here, and is really a bad practice.
Just instantiate the Map and put the entries after.